packages/block-library/src/cover/edit.native.js

```diff
@@ -278,20 +279,23 @@ const Cover = ( {
 		</TouchableWithoutFeedback>
 	);
 
-	const onChangeUnit = ( nextUnit ) => {
+	const onChangeUnit = useCallback( ( nextUnit ) => {
 		setAttributes( {
 			minHeightUnit: nextUnit,
 			minHeight:
 				nextUnit === 'px'
 					? Math.max( CONTAINER_HEIGHT, COVER_MIN_HEIGHT )
 					: CONTAINER_HEIGHT,
 		} );
-	};
+	}, [] );
 
 	const controls = (
 		<InspectorControls>
 			<OverlayColorSettings
-				attributes={ attributes }
+				overlayColor={ attributes.overlayColor }
+				customOverlayColor={ attributes.customOverlayColor }
```

Comment:
`customOverlayColor` is already declared here https://github.com/WordPress/gutenberg/pull/28235/files#diff-efe4dc27db844b16612461b3acc8ac6d6ee9c44a490baf9bb58c03e6f88b77d7R101

Maybe you could add the rest of the attributes there as well?

Reply:
I was thinking about that, however `overlayColor` comes from props and attributes as well, 
that's why I chose not to do destructuring to avoid confusion